plugins/@grouparoo/mailjet/src/lib/client.ts

```diff
@@ -125,7 +125,7 @@ class MailjetClient {
     const request = await this.client
       .post("contact", { version: "v3" })
       .request({
-        IsExcludedFromCampaigns: "true",
+        IsExcludedFromCampaigns: "false",
```

Comment:
This might be a good place for a Destination Option so you can choose how to handle `IsExcludedFromCampaigns` per destination. 

cc @pauloouriques

Reply:
For now, we can probably assume people want to have campaigns and do the option later.

Reply:
Agreed. I'll keep the Destination Option alternative in the loop.